install/Installer.sh

```diff
@@ -288,35 +248,16 @@ function configure_bash_completion() {
   
 }
 
-function get_user() {
-  local user 
-
-  user=$(who -m | awk '{print $1;}')
-  if [ -z "$YES" ]; then
-    if [ "$user" = "root" ] || [ -z "$user" ]; then
-      read -p "Please supply a non-root user to run Halyard as: " user
-    fi
-  fi
-
-  echo $user
-}
-
 function configure_halyard_defaults() {
-  home=$(getent passwd $HAL_USER | cut -d: -f6)
-  local halconfig_dir="$home/.hal"
-
-  echo "$(tput bold)Halconfig will be stored at $halconfig_dir/config$(tput sgr0)"
-
-  mkdir -p $halconfig_dir
-  chown $HAL_USER $halconfig_dir
+  echo "$(tput bold)Halconfig will be stored at /home/spinnaker/.hal/config$(tput sgr0)"
 
   mkdir -p /opt/spinnaker/config
   chmod +rx /opt/spinnaker/config
 
   cat > /opt/spinnaker/config/halyard.yml <<EOL
 halyard:
   halconfig:
-    directory: $halconfig_dir
+    directory: /home/spinnaker/.hal/config
```

Comment:
Wasn't `$halconfig_dir` set to `/home/spinnaker/.hal` before this PR?

Reply:
Nope, the variable was set using the command line options that I removed, which is why I removed the variable and hard-coded it instead, since the Debian package installs it to /home/spinnaker anyway.

Reply:
I was looking a few lines above at
```
local halconfig_dir="$home/.hal"
```
I'm all for hard-coding it.  I was really asking about the `/config` at the end since it seems like we didn't have it before.

Reply:
Fixed